tests/jballerina-unit-test/src/test/resources/test-src/types/table/table-negative.bal

```diff
@@ -477,3 +477,35 @@ function testMultipleKeys3() {
 
     Person1? _ = t1[1, 2, 3]; // error
 }
+
+type FooRec record {
+    readonly int x;
+    int y;
+};
+
+FooRec spreadField1 = {x: 1002, y: 30};
+FooRec spreadField2 = {x: 1003, y: 25};
+
+table<FooRec> key(x) tb1 = table [
+            {x: 1001, y: 20},
+            {...spreadField1},
+            {...spreadField2}
+        ];
+
+table<FooRec> key(x) tb2 = table [
+            {...spreadField1},
+            {...spreadField2}
+        ];
```

Comment:
Shall We follow same indentation like `FooRec` in L481.
ex:- 
```
table<FooRec> key(x) tb2 = table [
    {...spreadField1},
    {...spreadField2}
];
```

Reply:
Yes. @prakanth97 Intellij Idea will not format Ballerina code properly at the moment, Please use VS Code to format the code